src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationAdminResource.java

```diff
@@ -124,6 +124,23 @@ private String convertToJson(String yml) {
         return ResponseEntity.ok().build();
     }
 
+    @DeleteMapping(CONFIG + TENANTS + "/{tenant}")
+    @Timed
+    @PreAuthorize("hasPermission({'request': #paths}, 'CONFIG.ADMIN.DELETE.LIST')")
+    public ResponseEntity<Void> deleteConfigurations(@RequestBody(required = false) List<String> paths,
+                                                     HttpServletRequest request) {
+
+        List<String> nonNullPaths = Optional.ofNullable(paths)
+                                           .orElseGet(Collections::emptyList);
+
+        if (nonNullPaths.isEmpty()) {
+            configurationService.deleteConfiguration(extractPath(request));
+        } else {
+            configurationService.deleteConfigurations(nonNullPaths);
+        }
+        return ResponseEntity.ok().build();
```

Comment:
Maybe it will better to use only single call of configurationService?
`
String path=nonNullPaths;

if (nonNullPaths.isEmpty()) {

path = extractPath(request);

}
configurationService.deleteConfigurations(path)
`

Reply:
i think it will not work if you try to remove the tenant folder with reauest:
```
DELETE http://.../config/tenants/MYTENANT
```
that is because these two methods interferes: 
```
@DeleteMapping(CONFIG + TENANTS + "/{tenant}/**")
    public ResponseEntity<Void> deleteConfiguration(HttpServletRequest request) {
```

and 
```
    @DeleteMapping(CONFIG + TENANTS + "/{tenant}")
    public ResponseEntity<Void> deleteConfigurations(@RequestBody(required = false) List<String> paths,
                                                     HttpServletRequest request) {
```